src/expr/benches/like_pattern.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use expr::like_pattern;
+
+// SMART, by Shel Silverstein
+const POEM: &[&str] = &[
+    "My dad gave me one dollar bill",
+    "'Cause I'm his smartest son,",
+    "And I swapped it for two shiny quarters",
+    "'Cause two is more than one!",
+    //
+    "And then I took the quarters",
+    "And traded them to Lou",
+    "For three dimes -- I guess he don't know",
+    "That three is more than two!",
+    //
+    "Just then, along came old blind Bates",
+    "And just 'cause he can't see",
+    "He gave me four nickels for my three dimes,",
+    "And four is more then three!",
+    //
+    "And I took the nickels to Hiram Coombs",
+    "Down at the seed-feed store,",
+    "And the fool gave me five pennies for them,",
+    "And five is more than four!",
+    //
+    "And then I went and showed my dad,",
+    "And he got red in the cheeks",
+    "And closed his eye and shook his head--",
+    "Too proud of me to speak!",
+];
```

Comment:
This is a copyright violation, no?

Reply:
Booooooo.... But, yes, you're right. I will try to find something in the public domain.

Reply:
I was we hoping might get a @TonyRippy original work. :D